lib/Parse/ParseExpr.cpp

```diff
@@ -2513,16 +2513,16 @@ parseClosureSignatureIfPresent(SmallVectorImpl<CaptureListEntry> &captureList,
       // Create the VarDecl and the PatternBindingDecl for the captured
       // expression.  This uses the parent declcontext (not the closure) since
       // the initializer expression is evaluated before the closure is formed.
-      auto specifierKind = (ownershipKind != ReferenceOwnership::Weak)
-                               ? VarDecl::Specifier::Let
-                               : VarDecl::Specifier::Var;
+      auto specifierKind = optionalityOf(ownershipKind) == Optionality::Required
```

Comment:
This is specific to `weak`, not to general optionality.

Reply:
Okay. I can change that.